ui/src/alert-configuration/ConfigureAlerting.test.tsx

```diff
@@ -0,0 +1,1275 @@
+      },
+    });
+    await waitFor(() => {
+      return expect(
+        screen.getByText(/Redirected Alert Page/i),
+      ).toBeInTheDocument();
+    });
+    expect(screen.getByText(/Redirected Alert Page/i)).toBeInTheDocument();
+  });
+
+  it('test the "send a test email" button (NO AUTHENTICATION)', async () => {
+    await commonSetup();
+
+    await act(async () => {
+      await userEvent.type(
+        selectors.host(),
+        'smtp4dev.default.svc.cluster.local',
+      );
+      await userEvent.type(selectors.port(), '42');
+
+      await userEvent.type(selectors.sender(), 'renard.admin@scality.com');
+
+      await userEvent.type(
+        selectors.recipient(),
+        'user1@test.com, user2@test.com',
+      );
+
+      await userEvent.click(selectors.sendTestingEmailButton());
+    });
+
+    await waitForElementToBeRemoved(() => {
+      return screen.getByText(/Sending.../);
+    });
+
+    expect(saltLoginRequest).toHaveBeenCalledWith({
+      eauth: 'kubernetes_rbac',
+      username: 'oidc:renard.admin@scality.com',
+      token: 'xxx-yyy-zzz-token',
+    });
+
+    const data = `apiVersion: addons.metalk8s.scality.com
+kind: AlertmanagerConfig
+spec:
+  notification:
+    config:
+      route:
+        routes:
+          - group_by:
+              - ...
+            group_wait: 1s
+            group_interval: 1s
+            matchers:
+              - alertname = \"dummy_alert\"
+            receiver: test-receiver-config-from-ui
+      receivers:
+        - name: test-receiver-config-from-ui
+          email_configs:
+            - smarthost: smtp4dev.default.svc.cluster.local:2542
+              require_tls: false
+              from: renard.admin@scality.com
+              to: user1@test.com, user2@test.com
+              html: TEMPLATE DELETED
+`;
+
+    expect(patchAlertmanagerConfig).toHaveBeenCalledWith({
+      data: {
+        'config.yaml': data,
+      },
+    });
+    await waitFor(() => {
+      return expect(
+        screen.getByText(/The email has been sent, please check your email/i),
+      ).toBeInTheDocument();
+    });
+  });
+
+  it('test the "send a test email" button (LOGIN)', async () => {
+    await commonSetup();
+
+    await act(async () => {
+      await userEvent.type(
+        selectors.host(),
+        'smtp4dev.default.svc.cluster.local',
+      );
+      await userEvent.type(selectors.port(), '42');
+
+      await userEvent.click(selectors.authSelect.authClick());
+      await userEvent.click(selectors.authSelect.optionLogin());
+
+      await userEvent.type(selectors.username(), 'Renard');
+      await userEvent.type(selectors.password(), 'Renard Password');
+
+      await userEvent.type(selectors.sender(), 'renard.admin@scality.com');
+      await userEvent.type(
+        selectors.recipient(),
+        'user1@test.com, user2@test.com',
+      );
+
+      await userEvent.click(selectors.sendTestingEmailButton());
+    });
+
+    await waitForElementToBeRemoved(() => {
```

Comment:
Maybe add an additional check on the save button being disabled before this one

Reply:
Indeed :)